docs/reference-guide/query-modules/api/c-api.md

```diff
@@ -32,8 +32,11 @@ This is the API documentation for `mg_procedure.h` which contains declarations o
 |                | Name           |
 | -------------- | -------------- |
 | void * | **[mgp_alloc](#function-mgp_alloc)**(struct mgp_memory * memory, size_t size_in_bytes) <br/>Allocate a block of memory with given size in bytes.  |
+| void * | **[mgp_global_alloc](#function-mgp_global_alloc)**(size_t size_in_bytes) <br/>Allocate a block of memory with given size in bytes using global memory resource.  |
```

Comment:
If it's not a big deal can you escape all the asterisks? That way we will also get proper formatting on the webpage.